circle.yml

```diff
@@ -1,13 +1,19 @@
 dependencies:
+  pre:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash
   override:
     - printf '%s\n' color=false progress=false >.npmrc
     - rm -rf node_modules
```

Comment:
> I wonder do we need the `rm -rf node_modules` part of `circle.yml` now?

Let's try removing the line above. Also, the line above that is no longer necessary.